src/index.js

```diff
@@ -183,9 +219,16 @@ export default class Firedux {
   }
   logout () {
     const { dispatch } = this
+
     return new Promise((resolve, reject) => {
       dispatch({type: 'FIREBASE_LOGOUT_ATTEMPT'})
-      this.ref.unauth()
+      if (this.v3) {
+        this.auth().signOut()
+        .then(() => resolve(), (error) => reject(error))
```

Comment:
i'm considering those things with unauth placed below. i'll update with PR. would be glad to review it after that. thanks